.mockery.yaml

```diff
@@ -0,0 +1,6 @@
+packages:
+  github.com/nephio-project/nephio/controllers/pkg/giteaclient:
+    interfaces:
+      GiteaClient:
+        config:
+          dir: "{{.InterfaceDir}}"
```

Comment:
Not sure on the best way to fix this one but..
The mock file gets picked up by the coverage report. ie 0%
Using the config in mockery we can name the file with a _test suffix, but for me this breaks the import in the test.go.
Alternatively, which is what I plan to do for say the controller-runtime client mock, is to output to a "mocks" dir under controller/pkg and maybe exclude that in the report. Or we suffix them with _test and add the logic for that in the coverage cmd.
Thoughts?

Reply:
Can we take this discussion separately? Would it be OK to bring this PR in? I'll look at this in a subsequent PR.